src/components/x-img/index.vue

```diff
@@ -48,7 +49,11 @@ export default {
       defaut: 100
     },
     class: String,
-    scroller: Object
+    scroller: Object,
+    container: {
+      type: String,
+      default: 'window'
```

Comment:
There is no selector named `'window'`, there is only one `window`. Blazy will use `window` if no container is specified.
https://github.com/dinbror/blazy/blob/master/blazy.js#L54